SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyTests/DynamicActorAwakeAfterDormantChangePropertyTest.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "DynamicActorAwakeAfterDormantChangePropertyTest.h"
+
+#include "DormancyTestActor.h"
+
+// This test checks whether changes on a dormant actor are replicated when the actor becomes awake.
+
+ADynamicActorAwakeAfterDormantChangePropertyTest::ADynamicActorAwakeAfterDormantChangePropertyTest()
+{
+	Author = "Matthew Sandford";
+}
+
+void ADynamicActorAwakeAfterDormantChangePropertyTest::PrepareTest()
+{
+	Super::PrepareTest();
+
+	// Step 1 - Spawn dormancy actor and change NetDormancy to DORM_DormantAll
+	AddStep(TEXT("ServerSpawnDormancyActor"), FWorkerDefinition::Server(1), nullptr, [this]() {
+		AActor* Actor = CreateDormancyTestActor();
+		Actor->SetNetDormancy(DORM_DormantAll);
```

Comment:
What is the intended behaviour here, even though it's dormant, the server informs the client of the newly spawned actor?

Reply:
Yes, although that is not what is being tested here. 

The test then goes on to change a property on the actor. This should not be replicated as the actor is dormant. The actor is then awoken. The client should then receive the state of the property.